com.unity.render-pipelines.universal/Runtime/2D/LightUtility.cs

```diff
@@ -57,8 +59,17 @@ private struct SpriteLightMeshVertex
             };
         }
 
-        public static Bounds GenerateParametricMesh(Mesh mesh, float radius, float falloffDistance, float angle, int sides)
+        public static Bounds GenerateParametricMesh(Mesh mesh, float radius, float falloffDistance, float angle, int sides, LightCachedMeshData cachedMeshData)
         {
+
+            bool staticUpload = Application.isPlaying;
+#if !UNITY_EDITOR
+            staticUpload = true;
+#endif
```

Comment:
This is not necessary as `Application.isPlaying` always returns `true` in player builds.

Reply:
Fixed in 94f30ac326f1fc161502821c5d49373b19127b16